src/views/Ebic/index.jsx

```
@@ -17,7 +17,7 @@ import {
 } from "../../app/features/ebic";
 import EmojiCard from "../../components/card/EmojiCard";
 
-const Ebic = () => {
+const Ebic = ({ isDashboard = false }) => {
```

ESLint

[error] 'isDashboard' is missing in props validation

(react/prop-types)